ex03.ex

```diff
@@ -153,7 +173,15 @@ defmodule Ex03 do
   Think a little about a nice way to lay this code out.
   """
 
-  def won . . . "your code"
```

Comment:
# -1

I was hoping for a 2d layout

~~~ elixir
 def won({ m, m, m,
            _, _, _,
            _, _, _ }), do: m

  def won({ _, _, _,
            m, m, m,
            _, _, _ }), do: m
# . . .
~~~